lib/postcode_validation/gateway/pca_address_list.rb

```diff
@@ -0,0 +1,43 @@
+module PostcodeValidation
+  module Gateway
+    class PCAAddressList
+      include HTTParty
+
+      class PCARequestError < PostcodeValidation::Error::RequestError; end
+
+      KEY = ENV['POSTCODE_ANYWHERE_KEY']
+      base_uri 'https://services.postcodeanywhere.co.uk'
```

Comment:
Lot's of duplication here.  Not sure how to get rid of it, dependency inject something else? Surely inheritance is not the solution.